code/modules/food_and_drinks/food/candy.dm

```diff
@@ -134,14 +128,15 @@
 	trash = /obj/item/trash/candy
 	filling_color = "#7D5F46"
 	bitesize = 3
-	list_reagents = list("nutriment" = 2, "chocolate" = 5)
+	junkiness = 25
```

Comment:
Adressed this; all chef made bars are now a subtype of `confectionery`; reagent amounts have not changed, but they are no longer junk food.